cpp/src/dbscan/runner.h

```diff
@@ -49,14 +51,20 @@ __global__ void relabelForSkl(Index_* labels, Index_ N, Index_ MAX_LABEL) {
  * Turn the non-monotonic labels from weak_cc primitive into
  * an array of labels drawn from a monotonically increasing set.
  */
-template <typename Index_ = long>
+template <typename Index_ = int>
 void final_relabel(Index_* db_cluster, Index_ N, cudaStream_t stream) {
   Index_ MAX_LABEL = std::numeric_limits<Index_>::max();
   MLCommon::Label::make_monotonic(
     db_cluster, db_cluster, N, stream,
     [MAX_LABEL] __device__(Index_ val) { return val == MAX_LABEL; });
 }
 
+int64_t curTimeMillis() {
```

Comment:
maybe we could just add a `std::chrono` based timer logic inside `utils.h` and use it everywhere?

Reply:
Done.